pkg/tsdb/stackdriver/types.go

```diff
@@ -14,6 +14,42 @@ type (
 		GroupBys    []string
 		AliasBy     string
 		ProjectName string
+		Selector    string
+		Service     string
+		Slo         string
+	}
+
+	MetricQuery struct {
+		ProjectName        string
+		Unit               string
+		MetricType         string
+		CrossSeriesReducer string
+		AlignmentPeriod    string
+		PerSeriesAligner   string
+		GroupBys           []string
+		Filters            []string
+		AliasBy            string
+		MetricKind         string
+		ValueType          string
+		View               string
```

Comment:
I don't think that `Unit`, `MetricKind` and `ValueType` are used.

Reply:
👍